pokemon-app/src/App.tsx

```diff
@@ -1,12 +1,69 @@
-import React from 'react';
-import './App.css';
+import React, { useEffect, useState } from 'react';
```

Comment:
For better code readability please structure the imports as:
import react, {others} from "react"; // Core libraries

import {SomeComponent} from "./component"; // Your components
import {SomeComponent} from "./component";

import styles from "./styles.scss"; // Your styles